### DISCUSSION TITLE
deleted

### DISCUSSION THREAD

**Discussion by ghost** [julianxhokaxhiu/FFNx#182] [General]

**Comment by julianxhokaxhiu** [comment]
Thanks for the report but I've personally tested this case and works fine here. Most probably the XInput driver being used through 7h does not work as expected. I've tested this scenario with a DS4 Pad w/DS4Windows in XInput mode. Works fine. Other people have tested this with an XBox controller and works fine too.

Because of this I suggest you asking support in the relative 7th support channels :)

Cheers!

//EDIT: Also, to make sure you're really using the XInput mode, check the log file, it should mention this: https://github.com/julianxhokaxhiu/FFNx/blob/master/src/common.cpp#L789
